pkg/storage/bucket/sse_bucket_client_test.go

```diff
@@ -127,3 +127,9 @@ func (m *mockTenantConfigProvider) S3SSEKMSKeyID(_ string) string {
 func (m *mockTenantConfigProvider) S3SSEKMSEncryptionContext(_ string) string {
 	return m.s3KmsEncryptionContext
 }
+
+func secretWithValue(v string) flagext.Secret {
```

Comment:
I would upstream this to dskit. Looks a very handy utility.

Reply:
https://github.com/grafana/dskit/pull/155